balancer/weightedroundrobin/balancer_test.go

```diff
@@ -520,6 +508,8 @@ func (s) TestBalancer_TwoAddresses_BlackoutPeriod(t *testing.T) {
 	}
 }
 
+// Tests that the weight expiration period causes backends to use 0 as their
+// weight once the expiration period elapses.
```

Comment:
Can you specify what happens when address uses 0?

Reply:
Done